### DISCUSSION TITLE
No internet

### DISCUSSION THREAD

**Discussion by decenttp** [ltsp/ltsp#524] [Q&A] [answered]
Hi,

In a new Ubuntu Mate 20.04 installation in the chrootless setup, clients are unable to access internet. Internet is working fine on the LTSP server though. What's happening is the LTSP server (IP: 192.168.1.111) is being picked up as the Internet Gateway (IP 192.168.1.1) when the two are different. I can see that when the clients boot up. LTSP server is acting as the DHCP server currently and they are both on the same network connected to the the same switch. I tried to include the following parameter in dnsmasq configuration : 

dhcp-option=option:router,192.168.1.1 but didn't have an impact. 

Also the /etc/hosts file on the LTSP client when booted has : SERVER 192.168.1.111 entry which i don't believe should be there.

What can i do?

Thank you.

**Comment by alkisg** [comment]
Open a terminal and type `ip a` to see your network card interface name, e.g. enp2s0.
Then upload the output of the following command, both from the server and from an LTSP client:

```shell
sudo /usr/lib/klibc/bin/ipconfig -n enp2s0
```

**Comment by decenttp** [comment]
Server: 
IP-Config: eno1 hardware address e1:07:1b:ff:09:b1 mtu 1500 DHCP RARP

Client: 

IP-Conflg: enp3s0 hardware address c0:3f:d5:6c:b3:78 mtu 1500 DHCP RARP IP-ConfIg: enp3s0 complete (dhcp from 192.168.1.111):
address: 192.168.1.200 broadcast: 192.168.1.255	netmask: 255.255.255.0
gateway: 192.168.1.111	s0	: 8.8.8.8	dnsl : 8.8.4.4
rootserver: 192.168.1.111 rootpath: filename :

**Comment by alkisg** [comment, marked as answer]
Did you restart dnsmasq after setting the gateway option? Paste your /etc/dnsmasq.d contents.

**Reply by decenttp** [reply]
Thanks a lot. I had missed restarting dnsmasq. That did the trick.
